lib/Extractor/Candidates.cpp

```diff
@@ -158,6 +158,7 @@ Inst *ExprBuilder::makeArrayRead(Value *V) {
   unsigned Width = DL.getTypeSizeInBits(V->getType());
   APInt KnownZero(Width, 0, false), KnownOne(Width, 0, false);
   bool NonZero = false, NonNegative = false, PowOfTwo = false;
+  unsigned NumSignBits = 0;
```

Comment:
Yes, here it should be initialized to 1 because by default the return value of computeNumSignBits is 1.